challenge_6/python/mjuiuc/test.py

```diff
@@ -0,0 +1,52 @@
+import unittest
+import time
+from Solution import solution
+
```

Comment:
Is this much whitespace typical in Python? I would assume 1 newline would suffice. Thoughts on removing the extra newlines.